server/swap/swap.go

```diff
@@ -2118,6 +2200,7 @@ func (s *Swapper) handleRedeem(user account.AccountID, msg *msgjson.Message) *ms
 			s.rmLiveWaiter(user, msg.ID)
 			s.respondError(msg.ID, user, msgjson.TransactionUndiscovered,
 				fmt.Sprintf("failed to find redeemed coin %v", coinStr))
+			// Client should retry the redeem request, maybe even rebroadcast.
```

Comment:
`// TODO:`

Reply:
It's really a client requirement and this is just a note of what server expects of the client in this case.  Client actually does retry the redeem request, although there's no rebroadcast function, but the client-side error might prompt the user to check their node/wallet while client continues to retry.

Reply:
In case the tx waiter expires before the inaction deadline i.e. where `bTimeout` is greater than `txWaitExpiration`? Because with this change set, server responding with this error _might_ mean the server has waited "long enough" to find the tx and did not find it - where long enough is up till the inaction deadline.

Reply:
Super good point since now these expiration times go right up to inaction there's no real time window for the client to do anything anyway.  Will remove these comments, but we might consider tweaking this if the client gets more sophisticated with dealing with  this (e.g. check node for sync issues, instruct the user to nudge it, etc.).